build.gradle

```diff
@@ -27,7 +27,7 @@ ext {
   checkerFrameworkVersion = '3.21.4'
   checkstyleToolVersion = '8.33'
   commonsIOVersion = '2.11.0'
-  ideProbeVersion = '0.24.0'
+  ideProbeVersion = '0.24.0+6-f555e22a-SNAPSHOT'
```

Comment:
Published locally from my rebased version of https://github.com/VirtusLab/ide-probe/pull/179, commit hash will be different in your case